cluster/node-pools/worker-default/userdata.yaml

```diff
@@ -8,7 +8,7 @@ write_files:
     path: /etc/kubernetes/secrets.env
     content: |
       NODEPOOL_TAINTS={{if index .NodePool.ConfigItems "taints"}}{{.NodePool.ConfigItems.taints}}{{end}}
-      NODE_LABELS={{ .Values.node_labels }}{{if eq .NodePool.Profile "worker-spotio-ocean"}},spot.io=true{{end}},node.kubernetes.io/distro=ubuntu{{if index .NodePool.ConfigItems "labels"}},{{.NodePool.ConfigItems.labels}}{{end}}
+      NODE_LABELS={{ .Values.node_labels }}{{if or (eq .NodePool.Profile "worker-spotio-ocean") (eq .NodePool.Profile "worker-spotio")}},spot.io=true{{end}},node.kubernetes.io/distro=ubuntu{{if index .NodePool.ConfigItems "labels"}},{{.NodePool.ConfigItems.labels}}{{end}}
```

Comment:
Add `.NodePool.IsSpotIO()` to CLM maybe?

Reply:
Added here: https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/431